plugins/hydra_ray_launcher/setup.py

```diff
@@ -26,7 +26,7 @@
     install_requires=[
         "boto3==1.17.17",
         "hydra-core>=1.1.0.dev7",
-        "ray==1.2.0",
+        "ray==1.3.0",
```

Comment:
since we are bumping dependency, could you also bump the plugin version here
```
plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/__init__.py
```